gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/test/java/io/gravitee/rest/api/management/v2/rest/resource/integration/IntegrationResourceTest.java

```diff
@@ -312,4 +317,36 @@ public void should_return_400_when_missing_body() {
             assertThat(response).hasStatus(HttpStatusCode.BAD_REQUEST_400);
         }
     }
+
+    @Nested
+    class DeleteIntegration {
```

Comment:
There is a test missing to check the permission ;)

Reply:
good catch, added :)